docs/README.md

```diff
@@ -3,5 +3,3 @@
 This document provides pointers for those looking to make documentation changes for the Magma project
 
 - [Documentation Overview](https://docs.magmacore.org/docs/next/docs/docs_overview) for general documentation information
-- `make help` for specific commands
```

Comment:
You appear to have removed the `make help` command from `lte/gateway/Makefile` but not from `docs/Makefile`, while I think the line here is referring to the latter.

Reply:
I removed the change in the docs folder - unless you think it would be good to also get rid of that?

Reply:
I don't have a strong opinion really. I've never used these commands, though now I know they're there, they're kind of useful :stuck_out_tongue: Not sure whether they're useful enough to keep though. The advantage the docs help command has is that it's documented somewhere.

Reply:
I will keep it as it is for now, then the change only affects the `lte/gateway` file.